mobile_verifier/src/reward_shares.rs

```diff
@@ -230,6 +230,9 @@ impl MapperShares {
                         subscriber_id: mas.subscriber_id,
                         discovery_location_amount,
                         verification_mapping_amount,
+                        reward_override_entity_key: mas
+                            .reward_override_entity_key
+                            .unwrap_or("".to_string()),
```

Comment:
```suggestion
                            .unwrap_or_default(),
```
will result in an empty string